f2/src/theme/style.css

```diff
@@ -290,7 +290,6 @@ body {
 /*Checkboxes and Radio Buttons*/
 
 .custom-control {
-  height: 2.5rem;
```

Comment:
This breaks other styling. I agree this needs a CSS update but simply removing the height property from all elements that have the custom-control class is not the answer.

Also, since this needs an update it would be a good time to look into [Emotion](https://emotion.sh/docs/introduction) and [Styled-System](https://styled-system.com/getting-started) instead of CSS. This should make adjusting component styles easier in the future.

Reply:
May I ask which other styling does it break? I tested all the pages. Can you give me an example?

Reply:
This is a small example, but you can see the conditional element overlaps the checkbox. 
![overflow](https://user-images.githubusercontent.com/62246403/93805427-bfc39480-fc15-11ea-9fd9-a04b7bfed627.png)

Also, here you see the spacing is inconsistent.
![spacing](https://user-images.githubusercontent.com/62246403/93806237-dcac9780-fc16-11ea-860f-0e4a4ab84f96.png)

Reply:
The second example (mid-feedback), is still old CDS code (our new refactored code has not been merged yet). That's actually the correct way for arranging the space between checkboxes/radios (i.e., if there are more text, it should have more space, so the text of the first checkbox won't overlap with second one) 
Here is a test I did, if you put a long text (I just randomly put some text in for the first radio to make it longer)after a radio (same for checkbox)before removing height=2.5rem, it looks like this:
![longtextBefore](https://user-images.githubusercontent.com/68352206/93811543-48dec980-fc1e-11ea-9340-c5cded34638a.png)
After removing height=2.5rem, it looks like this. As you can see, this way is more responsive.
![longtextAfter](https://user-images.githubusercontent.com/68352206/93811455-2b116480-fc1e-11ea-933b-48471e2cc1ea.png)

Reply:
You are right about your first example, the conditional field is a bit closer to the checkbox than before, but you need to look carefully to find it. I took a screen shot for both before and after. But I can try to fix it if Hong complains:)
Before removing height=2.5rem, it looks like this:
![conditionalFieldBefore](https://user-images.githubusercontent.com/68352206/93814616-c3114d00-fc22-11ea-8c07-1f7969aad6ef.png)
After removing height=2.5rem, it looks like this:
![conditionalFieldAfter](https://user-images.githubusercontent.com/68352206/93814645-cf95a580-fc22-11ea-9331-4116dc3b9bdd.png)

Reply:
I know it's probably not the perfect solution (I am a beginner of CSS:)). But that does seem work better. Of course, if you have better solution, you are more than welcome:)

Reply:
You are right, without the height an option with very long text is more responsive. However, the above change does indicate an unintended change to another element (even if it appears small). To restore the spacing of the conditional field could you try the following:

- Change the morgin-top of .conditional-field to 0.5rem

- Add padding-top: 0.25rem to .conditional-field

Reply:
Sure, I'll try that.